test/src/unit-ChunkedBuffer.cc

```diff
@@ -820,6 +820,8 @@ TEST_CASE("ChunkedBuffer: Test shallow copy", "[ChunkedBuffer][shallow_copy]") {
   void* chunked_buffer2_chunk_0;
   CHECK(chunked_buffer1.internal_buffer(0, &chunked_buffer1_chunk_0).ok());
   CHECK(chunked_buffer2.internal_buffer(0, &chunked_buffer2_chunk_0).ok());
+  // this test fails 'debug' build config with vs20-17/-19, see
```

Comment:
Is this something we need to address before merging this PR?

Reply:
Nothing to be addressed before merging this PR, just a note pointing to explanation when others may encounter it in the future (as I did, again, in this development effort.)

Reply:
Only aware of https://github.com/TileDB-Inc/TileDB/issues/1939 mentioned in that comment,
created before I started at tiledb,
unaware of any internal story ticket

Reply:
https://app.clubhouse.io/tiledb-inc/story/9198/unit-chunkedbuffer-cc-test-dependent-on-non-guaranteed-compiler-returnvalueoptimization